src/setupWorker/stop/createStop.ts

```diff
@@ -10,5 +13,11 @@ export const createStop = (context: SetupWorkerInternalContext) => {
     context.worker?.postMessage('MOCK_DEACTIVATE')
     context.events.removeAllListeners()
     window.clearInterval(context.keepAliveInterval)
+    if (!options?.quiet) {
+      console.groupCollapsed(
```

Comment:
Currently, this is an unterminated console group. Unless there's a `console.groupEnd()` following it, it will embrace all logs that are coming after this message and group them in `[MSW] Mocking disabled.` group.

Instead, let's use plain `console.log`, since we have no additional information to group in this case.

Reply:
Alright :raised_hands: